src/test/Spec.hs

```diff
@@ -66,8 +66,10 @@ integrationTests = testGroup "Solidity Integration Testing"
       , ("echidna_all_sender didn't shrink optimally",     solvedLen 3        "echidna_all_sender")
       ] ++ (["s1", "s2", "s3"] <&> \n ->
         ("echidna_all_sender solved without " ++ unpack n, solvedWith (n, []) "echidna_all_sender"))
-  , testContract "basic/contractAddr.sol"        Nothing
-      [ ("echidna_addr failed",                  not . solved "echidna_addr") ]
+  , testContract "basic/contractAddr.sol" Nothing
+      [ ("echidna_address failed",                         solved "echidna_address") ]
+  , testContract "basic/contractAddr.sol" (Just "basic/contractAddr.yaml")
+      [ ("echidna_address failed",                         passed "echidna_address") ]
```

Comment:
I'm not sure if these two tests should have the same name?

Reply:
it would require another argument to `testContract`